app/views/users/_form.html.erb

```diff
@@ -0,0 +1,34 @@
+<%= form_for @user, html: { multipart: true } do |f| %>
+  <% if @user.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@user.errors.count, "error") %> prohibited this user from being saved:</h2>
+
+      <ul>
+      <% @user.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
```

Comment:
I could see lines 3-11 being reused in other forms you create. Maybe worth moving this into it's own partial. 

I say maybe because you usually hold off on refactoring until the repeated code comes up again. Just something to keep in mind.